index.js

```diff
@@ -16,6 +16,17 @@ if (!process.env.OPEN_QUAD_WORD) {
 
 let app = express();
 
+// Add JSONP support
+app.set('jsonp callback name', 'callback');
```

Comment:
**thought**: this appears to the default and may be unnecessary

Reply:
`callback` is the name that Esri seems to use. I've been testing with portal.

This is not default and is needed to make it work.

Reply:
I just added a test for jsonp and realized that it was already supporting it before these additions so I removed this commit.